cmd/internal/cli/build.go

```diff
@@ -158,6 +159,14 @@ var buildTmpdirFlag = cmdline.Flag{
 	EnvKeys:      []string{"TMPDIR"},
 }
 
+// SINGULARITY_DISABLE_CACHE
+var buildDisableCacheFlag = cmdline.Flag{
```

Comment:
I think we should initialize this hidden flag like this:
```
var buildDisableCacheFlag = cmdline.Flag{
	ID:           "buildDisableCacheFlag",
	Value:        &disableCache,
	DefaultValue: false,
	Name:         "disablecache",
	Hidden:       true,
	EnvKeys:      []string{"DISABLE_CACHE"},
}
```

As is, it looks like this is creating a flag with the name "", which could cause issues if we register two flags in this manner since they will have a name collision

Reply:
Does it even need to be a hidden flag? *I think not...*

Reply:
I think its fine to have this be exposed, but make sure to add usage info